plugins/inventory/nmap.py

```diff
@@ -34,6 +34,12 @@
             description: list of addresses to exclude
             type: list
             elements: string
+        port:
+            description:
+                - Only scan specific port or port range (C(-p)).
+                - "Ex: (C(22; 1-65535; U:53,111,137,T:21-25,80,139,8080,S:9))"
```

Comment:
```suggestion
                - "For example, you could pass C(22; 1-65535; U:53,111,137,T:21-25,80,139,8080,S:9)."
```
BTW, does this mean you want the user to write:
```yaml
- nmap:
    port:
      - 22; 1-65535; U:53
      - 111
      - 137
      - T:21-25
      - 80
      - 139
      - 8080
      - S:9
```
?

Reply:
I do not know why I forgot, but I could also add them into the actual Examples section as well to clarify.

Reply:
You can mention some quick examples here, but using the examples section is also a good idea.

```suggestion
                - "For example, you could pass C(22) for a single port, C(1-65535) for a range of ports, or C(U:53,111,137,T:21-25,80,139,8080,S:9) to check port 53 with UDP, ports 21-25 with TCP, port 9 with SCTP, and ports 111, 137, 80, 139, and 8080 with all protocols."
```
(Assuming that's what happening here.)

Reply:
Yes, that is good.  Thank you!  I'll also add to the example section to the PR as well.